fab/fabfile.py

```diff
@@ -145,6 +145,7 @@ def tag_commit(self):
             tagger={
                 'name': user.name,
                 'email': user.email,
+                'date': datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
Nice, makes sense that it'd be github that cares.  No need to update if this is working, I meant that more as an FYI.